server/codies.js

```diff
@@ -55,6 +55,7 @@ export default class Codies {
   constructor() {
     this.redTeam = [];
     this.blue_team = [];
+    this.isSpymaster = false;
```

Comment:
There's only one of these classes at a time. A single boolean doesn't make sense because all connected players are using the same object server side.

Reply:
just tested this. you're right, this doesn't work the way I thought it did. but since this is a purely server-side action which does a broadcast to all clients, and clients do not broadcast back, it actually works somehow. It's still shit code though so I'm going to close the PR and fold it into player & team support